docs/releasehistory.rst

```diff
@@ -1,15 +1,23 @@
 Release History
 ***************
 
-0.19.0
-======
+0.18.2 - Bugfix release
+=======================
 
-New features
+Enhancements
 ------------
+- ``SAMSSampler`` supports expanded ensemble simulations
+- ``SAMSSampler`` accepts multiple criteria for switching the stage
```

Comment:
I think something went wrong when resolving conflicts and these got moved in 0.18.2, which has been already released. Can you move these back to a new 0.19.0 section?